python_modules/dagster/dagster/_core/definitions/source_asset.py

```diff
@@ -152,6 +153,11 @@ class SourceAsset(ResourceAddable):
         partitions_def (Optional[PartitionsDefinition]): Defines the set of partition keys that
             compose the asset.
         observe_fn (Optional[SourceAssetObserveFunction]) Observation function for the source asset.
+        auto_observe_interval_minutes (Optional[float]): While the asset daemon is turned on, a run
+            of the observation function for this asset will be launched at this interval. `observe_fn`
+            must be provided.
+        freshness_policy (Optional[FreshnessPolicy]): A constraint specifying how often this asset
+            is expected to observe new data.
```

Comment:
As written, I think this could be interpreted ambiguously:
- How often the observation is expected to run
- How often the observed data is expected to change

We're talking about the latter, right? Is there a way to make that more clear?